mos.yml

```diff
@@ -12,7 +12,8 @@ filesystem:
 config_schema:
   - ["sntp", "o", {title: "SNTP settings"}]
   - ["sntp.enable", "b", true, {title: "Enable SNTP"}]
-  - ["sntp.server", "s", "time.google.com", {title: "Server address. If DHCP lease contains NTP server, tries that first."}]
+  - ["sntp.server", "s", "time.google.com", {title: "Server address"}]
+  - ["sntp.follow_dhcp_ntp", "b", false, {title: "If DHCP lease contains NTP server, tries that first."}]
```

Comment:
`sntp.use_dhcp_ntp_server`
and please make `true` the default to preserve current behavior.

Reply:
Done, though the whole point of this PR is to undo the undesirable change to behaviour introduced previously

Reply:
i think using local NTP is a sensible default for most use cases.
feel free to override the default in your project though.